schemas/2.4.0.json

```diff
@@ -616,6 +616,12 @@
         "operationId": {
           "type": "string"
         },
+        "security": {
```

Comment:
Would it make sense to extract this into a new definition called `SecurityRequirements`? Then you will just need to use it by `"$ref": "#/definitions/SecurityRequirements"`

Reply:
> Would it make sense to extract this into a new definition called `SecurityRequirements` or similar? Then you will just need to use it by `"$ref": "#/definitions/SecurityRequirements"`

Hi @smoya  - Its an array of SecurityRequirements. Not sure if just indicating an array of a complex schema warrants a new definition. It would apply to all similar constructs (tags for example). Not sure it serves much purpose.

Reply:
Fair enough 👍 I agree we would need to be consistent and lot of changes will need to be applied.